guides/getting-started/index.md

```diff
@@ -109,22 +113,38 @@ If you need to stop the Docker containers, you can use `yarn stop-prereqs`.
 These commands will vary across applications, so check out the README
 if you are looking at different Cardstack projects.
 
-### Start the front end application
+## Start the front end application
 
 Next, let's start the front end and see Cardstack in action.
 
 ```
 yarn start-ember
 ```
 
-Visit [http://localhost:4200](http://localhost:4200) to see the main Cardboard app in action:
-
-![screenshot of the Cardboard project](/images/cardboard-initial.png)
+Visit [http://localhost:4200](http://localhost:4200) to see the main app in action:
 
 Your newly added Card can be found at [http://localhost:4200/photographers/1](http://localhost:4200/photographers/1).
 
-## Next steps
+## Preview editing mode with the Edges
+
+The Card SDK provides an editing mode for Cards, called Edges.
+To see what it looks like, add the following to `cardhost/app/templates/application.hbs`:
```

Comment:
Why are we adding the 'mock-login' helpter to application.hbs? Should't we add it to isolated.hbs or embedded.hbs?

Reply:
I realized that if you put the mock-login anywhere else, there must already be a seed-data instance of the card in order to create new cards. if we put it in application.hbs, you don't need any seed data in order to start creating cards.